fixtures/cameo/q-spot-40-rgb.json

```diff
@@ -0,0 +1,318 @@
+  "meta": {
+    "authors": ["HS-Event"],
+    "createDate": "2022-11-06",
+    "lastModifyDate": "2022-11-06"
+  },
+  "links": {
+    "productPage": [
+      "https://www.cameolight.com/en/solutions/dj-musicians/static-lighting/spotlights/16459/q-spot-40-rgbw"
+    ],
+    "manual": [
+      "https://www.cameolight.com/en/downloads/file/id/694585978"
+    ],
+    "other": [
+      "https://www.cameolight.com/en/downloads/file/id/940935430"
+    ]
+  },
+  "physical": {
+    "dimensions": [151, 130, 168],
+    "weight": 2.75,
+    "power": 55,
+    "DMXconnector": "3-pin",
+    "bulb": {
+      "type": "LED",
+      "lumens": 780
+    },
+    "lens": {
+      "name": "Manual",
+      "degreesMinMax": [4.5, 25]
+    }
+  },
+  "availableChannels": {
+    "Dimmer": {
+      "fineChannelAliases": ["Dimmer fine"],
+      "capability": {
+        "type": "Intensity"
+      }
+    },
+    "Color Macros": {
+      "capabilities":[
+        {
+          "dmxRange": [0,5],
+          "type": "NoFunction"
+        },
+        {
+          "dmxRange": [6,13],
+          "type": "ColorPreset",
+          "comment": "Red"
+        },
+        {
+          "dmxRange": [14,21],
+          "type": "ColorPreset",
+          "comment": "Amber"
+        },
+        {
+          "dmxRange": [22,29],
+          "type": "ColorPreset",
+          "comment": "Yellow Warm"
+        },
+        {
+          "dmxRange": [30,37],
+          "type": "ColorPreset",
+          "comment": "Yellow"
+        },
+        {
+          "dmxRange": [38,45],
+          "type": "ColorPreset",
+          "comment": "Green"
+        },
+        {
+          "dmxRange": [46,53],
+          "type": "ColorPreset",
+          "comment": "Turquoise"
+        },
+        {
+          "dmxRange": [54,61],
+          "type": "ColorPreset",
+          "comment": "Cyan"
+        },
+        {
+          "dmxRange": [62,69],
+          "type": "ColorPreset",
+          "comment": "Blue"
+        },
+        {
+          "dmxRange": [70,77],
+          "type": "ColorPreset",
+          "comment": "Lavender"
+        },
+        {
+          "dmxRange": [78,85],
+          "type": "ColorPreset",
+          "comment": "Mauve"
+        },
+        {
+          "dmxRange": [86,93],
+          "type": "ColorPreset",
+          "comment": "Magenta"
+        },
+        {
+          "dmxRange": [94,101],
+          "type": "ColorPreset",
+          "comment": "Pink"
+        },
+        {
+          "dmxRange": [102,109],
+          "type": "ColorPreset",
+          "comment": "Warm White"
+        },
+        {
+          "dmxRange": [110,117],
+          "type": "ColorPreset",
+          "comment": "White"
+        },
+        {
+          "dmxRange": [118,125],
+          "type": "ColorPreset",
+          "comment": "Cold White"
+        },
+        {
+          "dmxRange": [126,127],
+          "type": "Effect",
+          "comment": "Color Jumping Stop",
+          "speed": "stop"
```

Comment:
Please could you check if this is the correct way to do this?

Reply:
This should be

```ts
"type": "Effect",
"effectPreset": "ColorJump",
"speed": "stop"
```